src/test/java/org/gridsuite/study/server/service/client/timeseries/TimeSeriesClientTest.java

```diff
@@ -41,48 +52,50 @@
 /**
  * @author Thang PHAM <quyet-thang.pham at rte-france.com>
  */
-public class TimeSeriesClientTest extends AbstractRestClientTest {
```

Comment:
`AbstractRestClientTest` was only used by `TimeSeriesClientTest` (because others client tests was migrate to `AbstractWireMockRestClientTest`), and it was only doing the start & stop of webmockserver, which was replaced by the extension of mockwebserver-junit5.

Done in 086428391652fada02b198c1e74b7bf4f0b33653.